RoadMap_SwiftUI/RoadMap_SwiftUI/View/MainTabView.swift

```diff
@@ -0,0 +1,49 @@
+// MainTabView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Главное представление таб вью
+struct MainTabView: View {
+    // MARK: - Public properties
+
+    var body: some View {
+        CustomTabView(selection: $mainTabViewModel.selection) {
+            MainView()
+                .myTabItem {
+                    TabItem(text: "", icon: Constants.Images.carIcon)
```

Comment:
```suggestion
                    TabItem(icon: Constants.Images.carIcon)
```
Удали из модели text, он нам не нужен